app/models/user.rb

```diff
@@ -3,7 +3,10 @@
 class User < ApplicationRecord
   has_secure_password
 
-  validates :name, presence: true
+  validates :name, presence: true # TODO: Change into full_name or seperate first and last name.
+  # TODO: Validate email format.
   validates :email, presence: true, uniqueness: { case_sensitive: false, scope: :provider }
-  validates :provider, presence: true
+  validates :provider, presence: true # TODO: Set a default value or remove the presence validation.
```

Comment:
I think provider is something that will need to be explicitly set, so I don't think there's a need for a default value

Reply:
Alright, I'll leave the defaulting this way until we define the explicit user domain https://github.com/bigbluebutton/greenlight/pull/3300/files#diff-ef20d40ddd2dd2a254d26fd4c59655cbed6b541c5f908b81125d09848c1dfeb6R19
I left the comment there because it felt weird but it makes sense that we'll change it.

Reply:
Done,